src/app/pokedex/pokedex-container.tsx

```diff
@@ -0,0 +1,36 @@
+import { Pokemon } from "@/models/pokemon";
```

Comment:
look into using usestate and useeffect. we dont really want the component to be rendered before the data is there

Reply:
I've refactored this to use state and effect. It's a little more complex but much more functional, and also better informs that loading is happening